electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -420,7 +420,8 @@ def append(
         source: str,
         capacity_reduction: float,
         fuel_type: str,
-        outage_type: OutageType,
+        outage_type: OutageType | None = None,
+        generator_id: str | None = None,
```

Comment:
Do we need the generator ID?

Is this something that would work on other regions/sources or is it ENTSOE specific?

Reply:
I think it might work in some other regions as well, we saw some regions in the US that would also report this.